tango/common/util.py

```diff
@@ -108,29 +108,32 @@ def import_module_and_submodules(package_name: str, exclude: Optional[Set[str]]
         package_name, base_path = resolve_module_name(package_name)
     else:
         base_path = Path(".")
+    base_path = base_path.resolve()
 
     if exclude and package_name in exclude:
         return
 
     importlib.invalidate_caches()
 
-    # For some reason, python doesn't always add this by default to your path, but you pretty much
-    # always want it when using `--include-package`.  And if it's already there, adding it again at
-    # the end won't hurt anything.
-    with push_python_path(base_path):
```

Comment:
@epwalsh I thought `push_python_path` is basically doing this with the context manager. But for some reason that doesn't work with multiprocessing?

Reply:
We can remove `push_python_path` from the code then, since it's not being used elsewhere.

Reply:
`push_python_path` temporarily adds the right path to `sys.path`. The problem is in this case is that the multiprocessing library @Tbabm is using needs to re-import some modules to deserialize objects, and at that point the needed path is no longer on `sys.path`.

Reply:
Removed: b16315c

Reply:
Got it.